pkg/cmd/server/start/node_args.go

```diff
@@ -104,13 +99,13 @@ func (args NodeArgs) BuildSerializeableNodeConfig() (*configapi.NodeConfig, erro
 }
 
 // defaultHostname returns the default hostname for this system.
-func defaultHostname() (string, error) {
+func defaultHostname() string {
 
 	// Note: We use exec here instead of os.Hostname() because we
 	// want the FQDN, and this is the easiest way to get it.
-	fqdn, err := exec.Command("hostname", "-f").Output()
+	fqdn, err := exec.Command("uname", "-n").Output()
 	if err != nil {
-		return "", fmt.Errorf("Couldn't determine hostname: %v", err)
+		glog.Fatalf("Couldn't determine hostname: %v", err)
```

Comment:
This should be handled by other code, not here.  We do not glog.Fatal in common code.